### DISCUSSION TITLE
Include Special Seasons in "Progress" When Enabled

### DISCUSSION THREAD

**Discussion by Treveo** [trakt/showly#918] [Ideas / Feature Requests]
Description:
Currently, when the "Special Seasons" setting is enabled, these seasons do not appear in the "Progress" section. For users who want to track all seasons, including specials, it would be beneficial if they were displayed in "Progress" as well.

Proposed Solution:
Ensure that enabling the "Special Seasons" setting automatically includes these seasons in the "Progress" section.

Provide clear visual distinction (e.g., a label or icon) for Special Seasons if needed.

Benefits:
More comprehensive progress tracking for users who watch both main and special seasons.

**Comment by jpsnyder** [comment]
I hit this issue when watching Pantheon. Netflix canceled the show after the first season, even though they have already made the second season. The second season is available on YouTube, so I can watch it, but this second season is marked as "Special" because it's not official I guess. It's annoying to have to search and manually mark the episodes as a watch them. 

Perhaps only if the user started to mark a "Special" episode as watched, it will then count it as progress?

**Reply by jpsnyder** [reply]
Alternatively, it would be nice if there was something like a "pin to top" feature. So I can pin a show to the top of the list if I'm working on watching the specials.

**Reply by Treveo** [reply]
> Alternatively, it would be nice if there was something like a "pin to top" feature. So I can pin a show to the top of the list if I'm working on watching the specials.

I think this is already in de app. Long press on a TV show in 'Progress' will show "Pin to top" option.

**Reply by jpsnyder** [reply]
Oh.. indeed there is. Thanks :)

**Reply by jpsnyder** [reply]
Oh wait, this doesn't completely solve the problem. There is no way from what I can tell to pin to top a show where I have completely watched the regular episodes. 

For instance, I have seen all of Doctor Who (2024), but there is a new Christmas special I want to watch. I wanted to pin it to the top to remind myself to watch it. The show isn't in the progress list because I have watched everything. I tried unchecking an episode so that it appears and then pinned it to the top. However, as soon as I recheck the episode as watched the listing disappears, even though it is pinned to the top.

I would say, maybe there should be an option to have specials count as progress, but I can see that becoming annoying for some shows that have a bunch a promos listed as specials. 

Maybe allow the user to pin a show to the top in other places besides the progress screen and ensure a show doesn't disappear if it has been pinned?

Alternatively, allow a user to mark individual shows to count specials as progress?

**Comment by Treveo** [comment]
@michaldrabik: Gentle bump, I'm still curious about what you think of this request and whether you'll take it on board.

**Reply by michaldrabik** [reply]
It is on the board but being honest currently it is a low priority feature for me.

**Comment by Treveo** [comment]
@michaldrabik Are you planning to implement this one soon? Maybe I'm going to try this one and pop you a PR.

**Comment by Treveo** [comment]
@michaldrabik Gentle bump, do you still plan to implement this? Or should we just close this one?

**Comment by Treveo** [comment]
I don't think it makes sense if it stays open, I close it.

**Comment by Treveo** [comment]
Made a PR: #1155
